packages/dx-scheduler-core/src/plugins/drag-drop-provider/helpers.ts

```diff
@@ -249,3 +249,28 @@ const updateMultipleResourceInfo: PureComputed<
   }
   return [cellResource.id];
 };
+
+export const appointmentDragged: PureComputed<
+  [SchedulerDateTime, SchedulerDateTime, SchedulerDateTime, SchedulerDateTime, any, any], boolean
+> = (start, startPrev, end, endPrev, groupingInfo, groupingInfoPrev) => {
+  if (moment(start as Date).isSame(startPrev as Date)
+      && moment(end as Date).isSame(endPrev as Date)
+      && checkAppointmentFields(groupingInfo, groupingInfoPrev)) {
+    return false;
+  }
+  return true;
+};
+
+const checkAppointmentFields: PureComputed<
```

Comment:
Seems that this name doesn't reflect the function purpose - we should specify that groupingInfo is tested here